tests/FactoryTrait.php

```diff
@@ -84,20 +84,22 @@ private function provideUsers(): iterable
             'username' => 'adminUser',
             'password' => 'adminUser123',
             'email' => 'adminUser@example.com',
+            'type' => 'Person',
         ];
 
         yield [
             'username' => 'JohnDoe',
             'password' => 'JohnDoe123',
             'email' => 'JohnDoe@example.com',
+            'type' => 'Person',
         ];
     }
 
-    private function createUser(string $username, string $email = null, string $password = null, $active = true, $hideAdult = true, $about = null): User
+    private function createUser(string $username, string $email = null, string $password = null, string $type, $active = true, $hideAdult = true, $about = null): User
```

Comment:
```suggestion
    private function createUser(string $username, string $email = null, string $password = null, string $type = 'Person', $active = true, $hideAdult = true, $about = null): User
```

This will set a default value as input parameter.